src/main/java/com/minecolonies/coremod/blocks/BlockConstructionTapeCorner.java

```diff
@@ -0,0 +1,335 @@
+        }
+
+        return this.getDefaultState().withProperty(FACING, enumfacing);
+    }
+
+    @Override
+    public int getMetaFromState(@NotNull final IBlockState state)
+    {
+        return state.getValue(FACING).getIndex();
+    }
+
+    /**
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @Override
+    @Deprecated
+    public boolean isFullCube(final IBlockState state)
+    {
+        return false;
+    }
+
+    @Override
+    public boolean isPassable(final IBlockAccess worldIn, final BlockPos pos)
+    {
+        return true;
+    }
+
+    /**
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @Override
+    @Deprecated
+    public AxisAlignedBB getBoundingBox(final IBlockState state, final IBlockAccess source, final BlockPos pos)
+    {
+        if (state.getValue(FACING).equals(NORTH))
+        {
+            return new AxisAlignedBB((float) N_START_COLLISION_X,
+                                      (float) BOTTOM_COLLISION,
+                                      (float) N_START_COLLISION_Z,
+                                      (float) N_END_COLLISION_X,
+                                      (float) HEIGHT_COLLISION,
+                                      (float) N_END_COLLISION_Z);
+        }
+        if (state.getValue(FACING).equals(WEST))
+        {
+            return new AxisAlignedBB((float) W_START_COLLISION_X,
+                                      (float) BOTTOM_COLLISION,
+                                      (float) W_START_COLLISION_Z,
+                                      (float) W_END_COLLISION_X,
+                                      (float) HEIGHT_COLLISION,
+                                      (float) W_END_COLLISION_Z);
+        }
+        if (state.getValue(FACING).equals(SOUTH))
+        {
+            return new AxisAlignedBB((float) S_START_COLLISION_X,
+                                      (float) BOTTOM_COLLISION,
+                                      (float) S_START_COLLISION_Z,
+                                      (float) S_END_COLLISION_X,
+                                      (float) HEIGHT_COLLISION,
+                                      (float) S_END_COLLISION_Z);
+        }
+        else
+        {
+            return new AxisAlignedBB((float) E_START_COLLISION_X,
+                                      (float) BOTTOM_COLLISION,
+                                      (float) E_START_COLLISION_Z,
+                                      (float) E_END_COLLISION_X,
+                                      (float) HEIGHT_COLLISION,
+                                      (float) E_END_COLLISION_Z);
+        }
+    }
+
+    /**
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @SideOnly(Side.CLIENT)
+    @Override
+    @Deprecated
+    public boolean shouldSideBeRendered(IBlockState blockState, IBlockAccess blockAccess, BlockPos pos, EnumFacing side)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1133)